core/templates/components/skill-selector/skill-selector.component.spec.ts

```diff
@@ -447,4 +447,25 @@ describe('SkillSelectorComponent', () => {
       })
     ]);
   });
+
+  it('should correctly filter untriaged skill summaries', () => {
+  // Prepare test data.
+    component.untriagedSkillSummaries = [
+      new SkillSummary('skill1', 'Skill One', 'en', 1, 2, 3, 123, 456),
+      new SkillSummary('skill3', 'Skill Three', 'en', 1, 6, 7, 565, 468)
+    ];
+    let activeSkillId = 'skill2';
+    let searchText = 'Skill';
+
+    // Perform the filtering.
+    let filteredSkills = component.searchInUntriagedSkillSummaries(searchText);
+
+    // Assertions.
+    expect(filteredSkills.length).toBe(2);
+
+    filteredSkills.forEach(skill => {
```

Comment:
I suggest you just assertEqual for the expected value of filteredSkills.

Reply:
> I suggest you just assertEqual for the expected value of filteredSkills.

Done, thanks!